app/views/practices/timeline.html.erb

```diff
@@ -39,24 +36,13 @@
                                             <%= tl.label :timeline, 'Time frame:', class: 'usa-label text-bold display-inline-block timeline-duration' %>&nbsp;<span>Type the time frame as Month # (e.g. Month 1, Months 2-3)</span>
                                             <%= tl.text_field :timeline, class: 'usa-input practice-input width-card timeline-input' %>
                                         </div>
-                                        <%= tl.fields_for :milestones do |m| %>
-                                          <%
-                                            m_id = m.object.id
-                                          %>
-                                          <%= m.hidden_field :position, class: 'milestone-position' %>
                                           <div class="grid-row width-full">
                                               <div>
-                                                  <%= m.label :description, 'Milestone:', class: 'usa-label text-bold display-inline-block' %>&nbsp;<span>Type a description of the milestone.</span>&nbsp;<span class="text-base-light milestone-character-count" <% if m_id.present? %> id="milestone_<%= m_id %>_character_count" <%end%>>(0/100 characters)</span>
-                                                  <%= m.text_area :description, class: 'usa-input practice-input height-10 milestone-textarea', 'counter-id': "milestone_#{m_id}_textarea" %>
+                                                  <%= tl.label :milestone, 'Milestone:', class: 'usa-label text-bold display-inline-block' %>&nbsp;<span>Type a description of the milestone.</span>&nbsp;<span class="text-base-light milestone-character-count" <% if tl_id.present? %> id="milestone_<%= tl_id %>_character_count" <%end%>>(0/100 characters)</span>
```

Comment:
Couldn't we get rid of this file since the timeline form is being driven by `app/views/practices/_timeline.html.erb`? Or am I missing something 😅

Reply:
Removed file.  Looking in to updating and fixing broken tests for timeline.

Reply:
styles have been updated on front end (thx Elisa) and on the PE side.